AppVeyor builds are broken

Starting from this one:

https://ci.appveyor.com/project/widelands-dev/widelands/builds/35025293

We did not change anything in our build environment for that commit, so something has changed in AppVeyor itself.

The last platform update of Appveyor was already on Aug 29th: https://www.appveyor.com/updates/

So most likely something is broken in the msys2 repo.  All builds fail with not beeing able to compile the test program/check for gcc:
`[00:08:33] -- The C compiler identification is GNU 10.2.0
[00:08:34] -- The CXX compiler identification is GNU 10.2.0
[00:08:34] -- Check for working C compiler: C:/msys64/mingw64/bin/gcc.exe
[00:08:34] -- Check for working C compiler: C:/msys64/mingw64/bin/gcc.exe -- broken
[00:08:34] CMake Error at C:/Program Files (x86)/CMake/share/cmake-3.16/Modules/CMakeTestCCompiler.cmake:60 (message):
[00:08:34]   The C compiler
[00:08:34] 
[00:08:34]     "C:/msys64/mingw64/bin/gcc.exe"
[00:08:34] 
[00:08:34]   is not able to compile a simple test program.
[00:08:34] 
[00:08:34]   It fails with the following output:
[00:08:34] 
[00:08:34]     Change Dir: C:/projects/widelands/build/CMakeFiles/CMakeTmp
[00:08:34]     
[00:08:34]     Run Build Command(s):C:/msys64/mingw64/bin/ninja.exe cmTC_0b0a7 && [1/2] Building C object CMakeFiles/cmTC_0b0a7.dir/testCCompiler.c.obj
[00:08:34]     [2/2] Linking C executable cmTC_0b0a7.exe
[00:08:34]     FAILED: cmTC_0b0a7.exe 
[00:08:34]     cmd.exe /C "cd . && C:\msys64\mingw64\bin\gcc.exe    CMakeFiles/cmTC_0b0a7.dir/testCCompiler.c.obj  -o cmTC_0b0a7.exe -Wl,--out-implib,libcmTC_0b0a7.dll.a -Wl,--major-image-version,0,--minor-image-version,0  -lkernel32 -luser32 -lgdi32 -lwinspool -lshell32 -lole32 -loleaut32 -luuid -lcomdlg32 -ladvapi32 && cd ."
[00:08:34]     collect2.exe: error: ld returned 5 exit status
[00:08:34]     ninja: build stopped: subcommand failed.`

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -34,7 +34,7 @@ install:
   - cmd: "bash --login -c \"pacman -Suuyy --noconfirm\""
   - cmd: "bash --login -c \"pacman -Q\""
   # Installed required libs
-  - cmd: "bash --login -c \"pacman --noconfirm -S mingw-w64-%MINGWSUFFIX%-ninja mingw-w64-%MINGWSUFFIX%-boost mingw-w64-%MINGWSUFFIX%-SDL2_ttf mingw-w64-%MINGWSUFFIX%-SDL2_mixer mingw-w64-%MINGWSUFFIX%-SDL2_image mingw-w64-%MINGWSUFFIX%-glew\""
+  - cmd: "bash --login -c \"pacman --noconfirm -S mingw-w64-%MINGWSUFFIX%-cmake mingw-w64-%MINGWSUFFIX%-ninja mingw-w64-%MINGWSUFFIX%-boost mingw-w64-%MINGWSUFFIX%-SDL2_ttf mingw-w64-%MINGWSUFFIX%-SDL2_mixer mingw-w64-%MINGWSUFFIX%-SDL2_image mingw-w64-%MINGWSUFFIX%-glew\""
 
 shallow_clone: false
 

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -32,7 +32,6 @@ install:
   # - cmd: "bash --login -c \"sed -i 7d C:/msys64/etc/pacman.d/mirrorlist.mingw32\""
   # - cmd: "bash --login -c \"sed -i 7d C:/msys64/etc/pacman.d/mirrorlist.mingw64\""
   - cmd: "bash --login -c \"pacman -Suuyy --noconfirm\""
-  - cmd: "bash --login -c \"pacman -Q\""
   # Installed required libs
   - cmd: "bash --login -c \"pacman --noconfirm -S mingw-w64-%MINGWSUFFIX%-cmake mingw-w64-%MINGWSUFFIX%-ninja mingw-w64-%MINGWSUFFIX%-boost mingw-w64-%MINGWSUFFIX%-SDL2_ttf mingw-w64-%MINGWSUFFIX%-SDL2_mixer mingw-w64-%MINGWSUFFIX%-SDL2_image mingw-w64-%MINGWSUFFIX%-glew\""
 

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -32,6 +32,7 @@ install:
   # - cmd: "bash --login -c \"sed -i 7d C:/msys64/etc/pacman.d/mirrorlist.mingw32\""
   # - cmd: "bash --login -c \"sed -i 7d C:/msys64/etc/pacman.d/mirrorlist.mingw64\""
   - cmd: "bash --login -c \"pacman -Suuyy --noconfirm\""
+  - cmd: "bash --login -c \"pacman -Q\""
   # Installed required libs
   - cmd: "bash --login -c \"pacman --noconfirm -S mingw-w64-%MINGWSUFFIX%-ninja mingw-w64-%MINGWSUFFIX%-boost mingw-w64-%MINGWSUFFIX%-SDL2_ttf mingw-w64-%MINGWSUFFIX%-SDL2_mixer mingw-w64-%MINGWSUFFIX%-SDL2_image mingw-w64-%MINGWSUFFIX%-glew\""
 
@@ -40,7 +41,7 @@ shallow_clone: false
 branches:
   only:
     - master
-    - static_link
+    - Test_appveyor
 
 build_script:
   - cmd: cd %APPVEYOR_BUILD_FOLDER%

